server.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+#
+import argparse
+import glob
+import json
+import logging
+import os
+import sys
+from http.server import BaseHTTPRequestHandler, HTTPServer
+from pathlib import Path
+
+import yaml
+from jinja2 import Template
+
+TARGET_NAMESPACE_LABEL = os.getenv("LABEL", "user.kubeflow.org/enabled")
+TEMPLATES_FOLDER = os.getenv("TEMPLATES_FOLDER", "./resources")
+PORT = os.getenv("PORT", 80)
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+handler = logging.StreamHandler(sys.stdout)
+handler.setLevel(logging.DEBUG)
+formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+handler.setFormatter(formatter)
+logger.addHandler(handler)
+
+
+def main(port: int, label: str, folder: str) -> None:
+    """
+    Runs a server for injecting resources into namespaces by given label
+    """
```

Comment:
Good docstring, but pretty sure it wouldn't pass muster with the docstring checks in ruff or pydocstyle using the google styleguide??

Reply:
I think this will get tidied up as soon as there's CI applying this repo's `tox -e lint`

Reply:
Added lint